kv/kvserver: TestLeasePreferencesDuringOutage failed

kv/kvserver.TestLeasePreferencesDuringOutage [failed](https://tanzanite.cluster.engflow.com/invocations/default/46945f4d-004c-44de-b718-1b75b80cf16a?testReportRun=2&testReportShard=26&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on release-25.1 @ [5580b75f60b1e8cc657b29c4b7ac057baaef7942](https://github.com/cockroachdb/cockroach/commits/5580b75f60b1e8cc657b29c4b7ac057baaef7942):


```
=== RUN   TestLeasePreferencesDuringOutage
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLeasePreferencesDuringOutage716692357
    test_log_scope.go:76: use -show-logs to present logs inline
    client_lease_test.go:1179: 
        	Error Trace:	pkg/kv/kvserver/client_lease_test.go:1179
        	            				pkg/kv/kvserver/client_lease_test.go:1183
        	            				pkg/kv/kvserver/client_lease_test.go:1216
        	Error:      	Received unexpected error:
        	            	heartbeat failed on epoch increment
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/liveness.init
        	            	  | 	pkg/kv/kvserver/liveness/liveness.go:98
        	            	  | runtime.doInit1
        	            	  | 	GOROOT/src/runtime/proc.go:7206
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:7173
        	            	  | runtime.main
        	            	  | 	GOROOT/src/runtime/proc.go:253
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (2) heartbeat failed on epoch increment
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestLeasePreferencesDuringOutage
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestLeasePreferencesDuringOutage716692357
--- FAIL: TestLeasePreferencesDuringOutage (5.35s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=26</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLeasePreferencesDuringOutage.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-49385

--- a/pkg/kv/kvserver/client_lease_test.go
+++ b/pkg/kv/kvserver/client_lease_test.go
@@ -1058,7 +1058,7 @@ func TestLeasePreferencesDuringOutage(t *testing.T) {
 	skip.UnderShort(t)
 	// The test has 5 nodes. Its possible in stress-race for nodes to be starved
 	// out heartbeating their liveness.
-	skip.UnderRace(t)
+	skip.UnderDuressWithIssue(t, 144457)
 
 	stickyRegistry := fs.NewStickyRegistry()
 	ctx := context.Background()

